src/libasr/codegen/wasm_assembler.h

```diff
@@ -168,6 +168,20 @@ void emit_import_fn(Vec<uint8_t> &code, Allocator &al,
     emit_u32(code, al, type_idx);
 }
 
+void emit_declare_mem(Vec<uint8_t> &code, Allocator &al,
+                     uint32_t min_no_pages, uint32_t max_no_pages = 0) {
+    if (max_no_pages) {
```

Comment:
To make it clear:
```suggestion
    if (max_no_pages > 0) {
```

Reply:
Got it. Thank you for the guidance. I will update it in upcoming commits.

Reply:
This has been fixed in [WASM: Make memory max_no_of_pages condition clearer](https://github.com/lcompilers/lpython/pull/1497/commits/e46b457040140170ea7e44723891fa013843863a).